llvm/test/DebugInfo/Generic/mem2reg-promote-alloca-1.ll

```diff
@@ -21,7 +21,7 @@
 ; CHECK: define dso_local void @fun(i32 %param)
 ; CHECK-NEXT: entry:
 ; CHECK-NEXT: #dbg_value(i32 %param, ![[PARAM:[0-9]+]], !DIExpression(),
-; CHECK-NOT: #dbg_value({{.*}}, ![[PARAM]]
+; CHECK-NEXT: #dbg_value({{.*}}, ![[PARAM]]
```

Comment:
This should test for the contents of the `#dbg_value` as well (i.e. the value and DIExpression), to ensure that it's producing the right operands.